worlds/hk/docs/setup_pt_br.md

```diff
@@ -0,0 +1,52 @@
+# Guia de configuração para Hollow Knight no Archipelago
+
+## Programas obrigatórios
+* Baixe e extraia o Lumafly Mod Manager (gerenciador de mods Lumafly) do [Site Lumafly](https://themulhima.github.io/Lumafly/).
+* Uma cópia legal de Hollow Knight.
+   * Versões Steam, Gog, e Xbox Game Pass do jogo são suportadas.
+   * Windows, Mac, e Linux (incluindo Steam Deck) são suportados.
+
+## Instalando o mod Archipelago Mod usando Lumafly
+1. Abra o Lumafly e confirme que ele localizou sua pasta de instalação do Hollow Knight.
+2. Clique em "Install (instalar)" perto da opção "Archipelago" mod.
+   * Se quiser, instale também o "Archipelago Map Mod (mod do mapa do archipelago)" para usá-lo como rastreador dentro do jogo.
+3. Abra o jogo, tudo preparado!
+
+### O que fazer se o Lumafly falha em encontrar a sua pasta de instalação
+1. Encontre a pasta manualmente.
+   * Xbox Game Pass:
+      1. Entre no seu aplicativo XBox e mova seu mouse em cima de "Hollow Knight" na sua barra da esquerda. 
+      2. Clique nos 3 pontos depois clique gerenciar.
+      3. Vá nos arquivos e selecione procurar. 
+      4. Clique em "Hollow Knight", depois em "Conteúdo", depois clique na barra com o endereço e a copie.
+   * Steam:
+      1. Você provavelmente colocou sua biblioteca Steam num local não padrão. Se esse for o caso você provavelmente sabe onde está.
+         . Encontre sua biblioteca Steam, depois encontre a pasta do Hollow Knight e copie seu endereço.
+         * Windows - `C:\Program Files (x86)\Steam\steamapps\common\Hollow Knight`
+         * Linux/Steam Deck - `~/.local/share/Steam/steamapps/common/Hollow Knight`
+         * Mac - `~/Library/Application Support/Steam/steamapps/common/Hollow Knight/hollow_knight.app`
+2. Rode o Lumafly como administrador e, quando ele perguntar pelo endereço do arquivo, cole o endereço do arquivo que você copiou.
+
+## Configurando seu arquivo YAML
+### O que é um YAML e por que eu preciso de um?
+Um arquivo YAML é a forma que você informa suas configurações do jogador para o Archipelago.
+Olhe o [guia de configuração basica de um multiworld](/tutorial/Archipelago/setup/en) aqui no site do Archipelago para aprender mais.
+
+### Onde eu consigo o YAML?
+Você pode usar o [pagina de configurações do jogador para Hollow Knight](/games/Hollow%20Knight/player-options) aqui no site  do Archipelago 
```

Comment:
Missing accent in "pagina" (página).

Also, it should say "Você pode usar *a* página de configurações do jogador para Hollow Knight".

Also remove 1 of the 2 spaces after " aqui no site" and before "do Archipelago".